lib/net.js

```diff
@@ -62,7 +62,7 @@ exports.connect = exports.createConnection = function() {
   const args = new Array(arguments.length);
   for (var i = 0; i < arguments.length; i++)
     args[i] = arguments[i];
-  // TODO: use destructuring when V8 is fast enough
+  // TODO (@joyeecheung) use destructuring when V8 is fast enough
```

Comment:
Super-nit, but none of the other `TODO`s in the codebase have an @ in them, the most standard style I can find is:

```
// TODO(joyeecheung): use ...
```